src/pieces/copilot/ask_command.py

```diff
@@ -33,16 +33,20 @@ def on_message(self, response:"QGPTStreamOutput"):
             if response.status == 'COMPLETED':
                 self.live.update(Markdown(self.final_answer), refresh=True)
                 self.live.stop()
- 
+
                 self.message_compeleted.set()
                 Settings.pieces_client.copilot.chat = BasicChat(response.conversation)
 
+                # There might be a cleaner way to do this using BaseWebsocket.close()
+                # But this works for now
+                os._exit(0)
```

Comment:
This will exist in the run command too right?

Reply:
> This will exist in the run command too right?

@Bishoy-at-pieces I only put it directly in the AskClass of `ask_command.py` . Do you want it in the run command too? if so which .py script is that?

Reply:
yeah you need to check first if Settings.run_in_loop if it is true